mreg_cli/commands/dhcp.py

```diff
@@ -23,6 +24,25 @@ def __init__(self, cli: Any) -> None:
         super().__init__(cli, command_registry, "dhcp", "Manage DHCP associations.", "Manage DHCP")
 
 
+def ipaddress_from_ip_arg(arg: str) -> IPAddress | None:
```

Comment:
Add a line about what can be raised? Also use EntityOwnershipMismatch when multiple hosts are found to use the IP? Something like:

```
:raises: InputFailure when the IP does not exist.
:raises: EntityOwnershipMismatch when the IP is in use by multiple hosts.
```